zephyr/subsys/object/Kconfig

```diff
@@ -3,12 +3,196 @@
 # Copyright (c) 2022 Legrand North America, LLC.
 # SPDX-License-Identifier: MIT
 
-if BACNETSTACK
+
+config BACNET_USE_SECTION_ITERABLE_OBJECT_TABLE
+	bool "Use the Section Iterable for the Object Table"
+	default y
+	help
+		Use the Section Iterable for the Object Table
 
 config BACNET_USE_DYNAMIC_DESCRIPTION
 	bool "Use dynalic allocate for objects descriptions"
-	default false
+	default b
 	help
 		Use dynalic allocate for objects descriptions
 
-endif # BACNETSTACK
+config BACOBJ_ACC_ENABLE
```

Comment:
Parameters BACNET_xxx_ENABLE will matter when it will be possible to add bacnet objects one at a time, i.e. after merging OSSBS-23.

Reply:
@Michail-Antropov We need to start all Kconfigs from this Zephyr module with the same prefix (BACNETSTACK_*) to minimize name collisions with alternate modules downstream users may also pull into their codebase.  Please change `BACOBJ_ACC_ENABLE` to `BACNETSTACK_OBJECT_ACC_ENABLE`.